tests/cpp_tests/test.conf

```diff
@@ -0,0 +1,171 @@
+# task type, support train and predict
+task = train
+
+
```

Comment:
```suggestion
```

Please avoid using more than one consecutive blank line in these `.conf` and `.config` files, to make them easier to read.